### PR TITLE
[Rollouts] Add integration test app

### PR DESCRIPTION
**Pull request by themiswang**
Adding integration test app
Right now added 4 targets:
- Neither Crash nor RC integration
- Either one of Crash or RC integration
- Both Crash and RC integration

To run the test app:
- gcert your local machine
- run `./generate_featureRolloutsTestsApp.sh`

Console app: https://firebase-console-ci.corp.google.com/u/0/project/feature-rollouts-test-app/settings/general/ios:com.firebase.config.featureRolloutsTestApp

#no-changelog

### CODE REVIEW

File: FirebaseRemoteConfig/Tests/FeatureRolloutsTestApp/Shared/CrashButtonView.swift [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import FirebaseCrashlytics
+import Foundation
+import SwiftUI
+
+struct CrashButtonView: View {
```
Comment (themiswang) [inline FirebaseRemoteConfig/Tests/FeatureRolloutsTestApp/Shared/CrashButtonView.swift:21]: Has a interface for Crash testing. For RC side will add later if needed.

Comment (themiswang) [reply FirebaseRemoteConfig/Tests/FeatureRolloutsTestApp/Shared/CrashButtonView.swift:21]: screen shot: https://screenshot.googleplex.com/3Y6EbtsbkEzyTHm

**Comment by danasilver** [comment]
Just jotting down some notes for future me:

Had to swap the Ruby version in `.ruby-version` to 2.7.6 (installed via rbenv).

Script is in FirebaseRemoteConfig so:
```
gcert
./FirebaseRemoteConfig/generate_featureRolloutsTestApp.sh
```

Then just ran the app in Xcode and everything worked 🎈 

Screenshot 2024-01-05 at 11 05 06 AM

**Review by danasilver — APPROVED** [review]

**Comment by themiswang** [comment]
> Just jotting down some notes for future me:
> 
> Had to swap the Ruby version in `.ruby-version` to 2.7.6 (installed via rbenv).
> 
> Script is in FirebaseRemoteConfig so:
> 
> ```
> gcert
> ./FirebaseRemoteConfig/generate_featureRolloutsTestApp.sh
> ```
> 
> Then just ran the app in Xcode and everything worked 🎈
> 
> Screenshot 2024-01-05 at 11 05 06 AM

In case anyone need a ruby version control, here is a blog for that: https://www.moncefbelyamani.com/how-to-install-xcode-homebrew-git-rvm-ruby-on-mac/

**Pull request merged**